bool switch
string topic
UInt64 hz
---
string info
bool success